bundles/org.openhab.binding.fenecon/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.fenecon</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: FENECON Binding</name>
+
```

Comment:
See https://github.com/openhab/openhab-addons/pull/17174#discussion_r1732757206 the connection tag is still missing

Reply:
I have added the connection tag in the addon.xml. I can't find a connection tag in the pom.xml in any other bundle. Can you link me to an example?

Reply:
Sorry, wrong comment on wrong file. Please ignore. :-/